controllers/app.controller.js

```diff
@@ -69,3 +70,17 @@ exports.patchArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  if (isNaN(comment_id)) {
+    return next({ status: 400, msg: 'Invalid comment_id' });
+  }
```

Comment:
Error handling like this should not be done in the controller like this, and this does not need to be checked. This if left and allowed to make it to the query psql will throw an error for you